Open Judge System/OJS.Services/Business/Problems/ProblemsBusinessService.cs

```diff
@@ -139,12 +139,12 @@ public ServiceResult CopyToContestByIdByContestAndProblemGroup(int id, int conte
                 return new ServiceResult(CannotCopyProblemInActiveContestErrorMessage);
             }
 
-            this.CopyProblem(problem, contestId, problemNewOrderBy, problemGroupId);
+            this.CopyProblemToContest(problem, contestId, problemNewOrderBy, problemGroupId);
 
             return ServiceResult.Success;
         }
 
-        private void CopyProblem(Problem problem, int contestId, int newOrderBy, int? problemGroupId)
+        private void CopyProblemToContest(Problem problem, int contestId, int newOrderBy, int? problemGroupId)
```

Comment:
Rename according to convention. You don't need to specify that what you're copying is a `problem` since this is located in the `ProblemsBusinessService` and it's implied. Rename to simply `CopyToContest`